pages/admin/organisationen/KlasseCreationSuccess.page.ts

```diff
@@ -0,0 +1,36 @@
+import { expect, type Locator, Page } from '@playwright/test';
+import { AbstractAdminPage } from '../../abstracts/AbstractAdminPage.page';
+import { KlasseCreationParams } from './KlasseCreationView.neu.page';
+
+export class KlasseCreationSuccessPage extends AbstractAdminPage {
+  /* add global locators here */
+
+  constructor(page: Page) {
+    super(page);
+  }
+
+  /* actions */
+  async waitForPageLoad(): Promise<void> {
+    return this.page.getByTestId('klasse-success-text').waitFor({ state: 'visible' });
+  }
```

Comment:
```suggestion
public async waitForPageLoad(): Promise<void> {
  await this.page.getByTestId('klasse-success-text').waitFor({ state: 'visible' });
}```
returning here will just return the promise back.